services/frontend/.env

```diff
@@ -1,3 +1,4 @@
 REACT_APP_API_URL=https://api.eosio.cr
```

Comment:
@tetogomez  The smart contract is validating ratings are only being saved for registred block producers. Sinc the contract is being tested on the jungle testnet we need to change this variable to use the jungle testnet endpoint.

https://jungle.eosio.cr